skins/cat17/address-form/src/__tests__/store.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { getters } from '../store'
+import {AddressState, Validity} from '../types';
+
+function newMinimalStore( overrides: Object ): AddressState {
+	return Object.assign(
+		{
+			isValidating: false,
+			form: {
+				salutation: Validity.INCOMPLETE,
+				title: Validity.INCOMPLETE,
+				firstName: Validity.INCOMPLETE,
+				lastName: Validity.INCOMPLETE,
+				companyName: Validity.INCOMPLETE,
+				street: Validity.INCOMPLETE,
+				postcode: Validity.INCOMPLETE,
+				city: Validity.INCOMPLETE,
+				country: Validity.VALID,
+				addressType: Validity.VALID
+			}
+		},
+		overrides
+	);
+}
+
+describe('Store', () => {
+
+	it('returns no invalid fields on render', () => {
```

Comment:
The phrase "on render" is confusing to me, since the store itself does not render anything, we're only using the getters in the render part of the Vue component, but that is independent from the store.

Maybe something like "it returns no invalid fields" or "it returns no invalid field names when internal state does not contain them".

You could also add another level of `describe` for each setter.

Reply:
Right, "on render" is incorrect indeed.
> You could also add another level of `describe` for each setter.

Yep will do.